backend-rust/src/graphql_api/baker.rs

```diff
@@ -127,17 +129,52 @@ impl Baker {
             .transpose()?
             .map(|c| AmountFraction::new_unchecked(c).into());
 
+        let total_stake: i64 =
+            sqlx::query_scalar!("SELECT total_staked FROM blocks ORDER BY height DESC LIMIT 1")
+                .fetch_optional(pool)
+                .await?
+                .ok_or(ApiError::NotFound)?;
+
+        let row = sqlx::query!(
+            "
+                SELECT
+                    COUNT(*) AS delegator_count,
+                    SUM(delegated_stake)::BIGINT AS delegated_stake
+                FROM accounts 
+                WHERE delegated_target_baker_id = $1
+            ",
+            self.id.0
+        )
+        .fetch_one(pool)
+        .await?;
+
+        let delegated_stake = row.delegated_stake.unwrap_or(0);
+
+        // The total amount staked in this baker pool includes the baker stake
+        // and the delegated stake.
+        let total_pool_stake = self.staked + delegated_stake;
+
+        // Division by 0 is not possible because `total_staked` is always a positive
+        // number.
+        let total_stake_percentage = (total_pool_stake as f64 * 100.0 / total_stake as f64)
```

Comment:
Since we expose the result as a decimal, maybe we should use decimals for the computation as well, although it is way overkill for a percentage.
It just seems misleading to call it decimal in the API otherwise.